### ISSUE TITLE
Is JsTree available on npm?

### ISSUE THREAD

**Issue by vanillacandy** [adamjimenez/jstree-table#30]
I can't find the package on npm, can it be available on npm?

**Comment by mbroadhead** [comment]
As far as i'm aware, it is not available on the npm registry. You can still install it directly from github:
```
npm install https://github.com/adamjimenez/jstree-table
```

**Comment by adamjimenez** [comment]
This is correct, I'm trying to publish it but someone forked my repo and nicked the name.
